packages/interface/src/components/index.js

```diff
@@ -6,8 +6,4 @@ export { default as PinnedItems } from './pinned-items';
 export { default as MoreMenuDropdown } from './more-menu-dropdown';
 export { default as MoreMenuFeatureToggle } from './more-menu-feature-toggle';
 export { default as ActionItem } from './action-item';
-export { default as PreferencesModal } from './preferences-modal';
-export { default as PreferencesModalTabs } from './preferences-modal-tabs';
-export { default as PreferencesModalSection } from './preferences-modal-section';
-export { default as ___unstablePreferencesModalBaseOption } from './preferences-modal-base-option';
```

Comment:
I'm guessing these components were only here because we wanted them private. I've moved them to the preferences package as private APIs instead.